hydromt/workflows/region.py

```diff
@@ -112,14 +112,15 @@ def parse_region_basin(
     logger : Logger, optional
         Logger object.
     """
-    kwargs = region.copy()
+    var_thresh_kwargs = region.copy()
     kind = next(iter(region))
-    value0 = kwargs.pop(kind)
+    value0 = var_thresh_kwargs.pop(kind)
```

Comment:
Is the ordering of the region important?

Reply:
yes

Reply:
in the builtin module `collections` is an `OrderedDict`, maybe we can use that one? But that is nitpicky, we can refactor later if needed.

Reply:
Yes I agree. I'll make an ticket for it.